src/index.js

```diff
@@ -258,14 +351,30 @@ export default class ToggleBlock {
    */
   _iterateOnItems(items, toggleIndex) {
     let index = toggleIndex;
-    if (this.data.status === 'closed') {
-      for (let i = 0; i < items; i += 1) {
-        this.api.blocks.getBlockByIndex(index += 1).holder.setAttribute('hidden', true);
-      }
-    } else {
-      for (let i = 0; i < items; i += 1) {
-        this.api.blocks.getBlockByIndex(index += 1).holder.removeAttribute('hidden');
-      }
+
+    switch (this.data.status) {
+      case 'closed':
+        if (items > 0) {
+          for (let i = 0; i < items; i += 1) {
+            this.api.blocks.getBlockByIndex(index += 1).holder.setAttribute('hidden', true);
+          }
+        } else {
+          this.wrapper.lastChild.setAttribute('hidden', true);
+        }
```

Comment:
@juliansg022 IMO, you can extract this block of code and reuse it when the status is 'open'

Reply:
@juanhurtado10 I'll try it, but I was checking and now I think there might be a way to use the same code block in both cases.